### ISSUE TITLE
Issues downloading release with OniVim

### ISSUE THREAD

**Issue by JesseKPhillips** [Pure-D/serve-d#119]
This may not at all be an issue with the plugin, I am utilizing OniVim 0.6.x Nightly (https://onivim.io/) which is in development to support Visual Studio Code plugins. As such this is more likely an issue there than it is here and can be closed as won't fix. But it may be good reference for others.

After installation and restart
* I receive Syntax Highlighting for D files
* A notification pops up and informs me "Failed to download release"
* A "Compile from source" button is provided but appears to do nothing.

**Comment by WebFreak001** [comment]
so are you trying to load in code-d? That part is handled by code-d.

Try searching for extension debug logs (in vscode that is in Ctrl-Shift-P -> Developer Tools), otherwise I don't know what's broken.

**Comment by Airbus5717** [comment]
Onivim is still not stable

**Comment by JesseKPhillips** [comment]
Correct, onivim is still developing the Plugin support. I will try to get additional information from the developer tools. (taking this to onivim as well)

**Comment by JesseKPhillips** [comment]
Onivim doesn't provide developer tools, I'll need to find another way to get at error information.
